### ISSUE TITLE
Interoperability with "new" ST app?

### ISSUE THREAD

**Issue by dlasher** [sgupta999/mqtt-bridge-smartthings#32]
Any testing with the "new" ST App everyone is being forced to migrate to?

**Comment by jfgibbins** [comment]
ya, the testing is fubar.  Samsung's new and improved app has added nothing new, but broke a lot of things that did work, like this one.

**Comment by jfgibbins** [comment]
btw, didn't mean the code owners testing, I met the results from my testing of it after being forced onto the new smartthings...it went all fubar :(

**Comment by koensayr** [comment]
So it definitely doesn't work with the new? I had this working but I couldn't remember how I got it to actually work. Now I just think I'm spinning in circles

**Comment by koensayr** [comment]
Any thoughts here @sgupta999 or have you really moved on from this project?

**Comment by sgupta999** [comment]
I don’t really know if it works or not - like I said my whole environment is down. Some of the other device handlers have some features that work and some that don’t.

I haven’t looked at anything for a while

**Comment by koensayr** [comment]
I'm not sure entirely how yet. But I can comment that this codebase does seem to be working with the new Smartthings (ST) app

**Comment by koensayr** [comment]
@sgupta999 I think this issue at least can be closed

**Comment by koensayr** [comment]
agreed. I think it can be closed. But I think perhaps you forgot to close it @sgupta999
